src/NServiceBus.Core/Recoverability/RecoverabilityComponent.cs

```diff
@@ -18,7 +18,9 @@ class RecoverabilityComponent
         public RecoverabilityComponent(SettingsHolder settings)
         {
             this.settings = settings;
-
+            var configuration = settings.Get<Configuration>();
```

Comment:
The other components takes this as an argument to not use the settings inside the component but since this component is not yet aligned we should do that in a separate PR

Reply:
yeah that's the reason I did it this way for now